shell/platform/fuchsia/flutter/tests/session_connection_unittests.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "gtest/gtest.h"
+
+#include <lib/async-loop/default.h>
+#include <lib/sys/cpp/component_context.h>
+#include <lib/ui/scenic/cpp/view_token_pair.h>
+
+#include <fuchsia/sys/cpp/fidl.h>
+#include <fuchsia/ui/policy/cpp/fidl.h>
+
+#include "flutter/shell/platform/fuchsia/flutter/logging.h"
+#include "flutter/shell/platform/fuchsia/flutter/runner.h"
+#include "flutter/shell/platform/fuchsia/flutter/session_connection.h"
+
+using namespace flutter_runner;
+
+namespace flutter_runner_test {
+
+class SessionConnectionTest : public ::testing::Test {
```

Comment:
Is this hermetic?  We are using the actual system Scenic here.  Let's file an issue to track making this hermetic via TestWithEnvironment (for the hackathon)

Reply:
Filed issue fxb/44542